File: week-04/day-2_Classes/src/com/greenfox/Animal.java
package com.greenfox;

public class Animal {
    /* Animal
     Create an Animal class
     Every animal has a hunger value, which is a whole number
     Every animal has a thirst value, which is a whole number
     when creating a new animal object these values are created with the default 50 value
     Every animal can eat() which decreases their hunger by one
     Every animal can drink() which decreases their thirst by one
     Every animal can play() which increases both by one*/
    public static void main(String[] args) {
        Animal cat = new Animal();
        Animal dog = new Animal();

        //Test:
        System.out.println(cat.hunger);
        cat.eat();
        System.out.println(cat.hunger);
        cat.play();
        System.out.println(cat.hunger+ " "+cat.thirst);
    }

    int hunger = 50;
    int thirst = 50;

    public void eat() {
        hunger--;
    }

    public void drink() {
        thirst--;
    }

    public void play() {
        hunger++;
        thirst++;
    }


}

File: C#/classes/Animal.cs
using System;
class Animal
{
    public int Hunger { get; set; }
    public int Thirst { get; set; }

    public Animal()
    {
        this.Hunger = 50;
        this.Thirst = 50;

    }

    public void Eat()
    {
        this.Hunger--;
    }
    public void Drink()
    {
        this.Thirst--;
    }
    public void Play()
    {
        this.Hunger++;
        this.Thirst++;
    }
}